ios/RNFingerprintjsPro.swift

```diff
@@ -62,7 +62,7 @@ class RNFingerprintjsPro: NSObject {
         }
 
 
-    private static func parseRegion(_ passedRegion: String?, endpoint: String?) -> Region {
+    private static func parseRegion(_ passedRegion: String?, endpoint: String?, fallbackEndpointUrls: [String]?) -> Region {
```

Comment:
For naming consistency:
```suggestion
    private static func parseRegion(_ passedRegion: String?, endpoint: String?, endpointFallbacks: [String]) -> Region {
```
Also, the added parameter does not need to be an optional.